microsoft-edge/toc.yml

```diff
@@ -1029,89 +1046,167 @@
 
             - name: Publish a Microsoft Edge extension by using a GitHub account
               href: extensions-chromium/publish/github.md
 
-            - name: Add users to the Microsoft Edge program
-              href: extensions-chromium/publish/aad-account.md
-
             - name: Publish a Microsoft Edge extension
               href: extensions-chromium/publish/publish-extension.md
 
+#           - name: Extension sharing offline
+#             href:
+
             - name: Extension hosting
               href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
 
-            - name: Update a Microsoft Edge extension
-              href: extensions-chromium/publish/update-extension.md
-
-            - name: Automatically update extensions in Microsoft Edge
-              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
-
             - name: Submission states for extensions in the Microsoft Edge Add-ons website
               href: extensions-chromium/publish/submission-states.md
 
+            - name: Manage account settings
+              href: extensions-chromium/publish/manage-settings.md
+
+            - name: Alternative ways to distribute extensions
+              href: extensions-chromium/developer-guide/alternate-distribution-options.md
+
             - name: Curation and review process for the Edge Add-ons store
               href: extensions-chromium/publish/add-ons-store-curation.md
 
+            - name: REST API for publishing  # todo: if this is the only way to publish extension, move up?
+              items:
+                - name: Using the Microsoft Edge Add-ons REST API  # todo: See Also: APIs 101, JS APIs article
+                  href: extensions-chromium/publish/api/using-addons-api.md
+
+                - name: REST API Reference for Microsoft Edge Add-ons
+                  href: extensions-chromium/publish/api/addons-api-reference.md
+
+# -------------------------------------
+        - name: Manage and maintain an extension
+          items:
+            - name: Update a Microsoft Edge extension
+              href: extensions-chromium/publish/update-extension.md
+
+            - name: Automatically update an extension in Microsoft Edge  # todo: update top-of-article title
+              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
+
             - name: View Microsoft Edge extensions analytics dashboard
               href: extensions-chromium/publish/extensions-analytics.md
 
             - name: Reply to user reviews
               href: extensions-chromium/publish/reply-user-reviews.md
-              displayName: Respond to user reviews
+              displayName: Reply to user reviews about your Microsoft Edge extension, Respond to user reviews  # top-of-page title, alt title word  # todo: make top-of-title identical to toc title, clarify in first sentence
```

Comment:
disparity: toc title:
Reply to user reviews
top-of-article title:
Reply to user reviews about your Microsoft Edge extension

either:
* Make TOC title longer eg "Reply to user reviews about your extension"
* Make top-of-article shorter eg "Reply to user reviews about your Microsoft Edge extension" or "Reply to user reviews" (& change in all the usual spots).
* Either way, make sure first sentence clarifies the article title.

Reply:
globally changed title to "Reply to user reviews of an extension".
in first sentence, added "of your extension":
You can view and respond to user reviews of your extension that are posted on the Microsoft Edge Add-ons product page.